cartographer_grpc/handlers/add_landmark_data_handler_test.cc

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2018 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer_grpc/handlers/add_landmark_data_handler.h"
+#include "cartographer_grpc/testing/add_data_handler_test.h"
+#include "cartographer_grpc/testing/test_helpers.h"
+#include "google/protobuf/text_format.h"
+#include "gtest/gtest.h"
+
+namespace cartographer_grpc {
+namespace handlers {
+namespace {
+
+using ::testing::_;
+using ::testing::Eq;
+using ::testing::Pointee;
+using ::testing::Truly;
+
+const std::string kMessage = R"PROTO(
```

Comment:
in google3 I think we would have "constexpr char kMessage[] = ", but we are not in google3 :)

Reply:
:)